clients/fides-js/src/lib/consent-types.ts

```diff
@@ -117,6 +117,9 @@ export interface FidesInitOptions {
 
   // Defines default primary color for consent components, but can still be overridden with overrides or custom CSS
   fidesPrimaryColor: string | null;
+
+  // Whether the developer forced the inclusion of the GPP extension
+  forceGpp: boolean;
```

Comment:
Is this the only var in  `FidesInitOptions` that cannot be set via an .env var? If so, can we comment this?

I think we should also comment that it can only be set via an explicit query param.

Reply:
technically, it's the only option that can be set as a query param on the `script` tag itself. I discussed this with Michael and going to include it as a section on the FidesOptions documentation. Great callout.

Reply:
ah gotcha, thanks for including in the docs!